src/common/getQueryParameters.js

```diff
@@ -0,0 +1,13 @@
+export default function getQueryParameters(search = global.location.search) {
+  const keyValueFragments = search
+    .slice(search.indexOf('?') + 1)
+    .split('&')
```

Comment:
Unfortunately this will break if there's an &amp; or any other URL encoded entity in the query string... 

Wondering if we can use one of the two approaches described here? https://davidwalsh.name/query-string-javascript

Reply:
(I know that the particular use case we have for this function won't care about the above, but if we're going to have a general-purpose function, then we should probably consider it)